games/arma3/entrypoint.sh

```diff
@@ -392,10 +392,16 @@ if [[ ${HC_NUM} > 0 ]]; then
     done
 fi
 
+# Define the log file path with a timestamp
+LOG_FILE="/home/container/logs/arma3server_$(date '+%m_%d_%Y_%H%M%S').log"
+
+# Ensure the logs directory exists
+mkdir -p /home/container/logs
+
 # Start the Server
 echo -e "\n${GREEN}[STARTUP]:${NC} Starting server with the following startup command:"
 echo -e "${CYAN}${modifiedStartup}${NC}\n"
-${modifiedStartup}
+${modifiedStartup} 2>&1 | tee -a "$LOG_FILE"
```

Comment:
If it's not too hard, would it be possible to detect if `-nologs` is present in `$STARTUP_PARAMS` and not tee logs to file if it is (simple IF statement)? Feel free to skip this if you don't have time; I can always implement it later 👍

Reply:
Yep, I can get this implemented.